app/src/main/menus.js

```diff
@@ -11,6 +12,8 @@ const checkForUpdatesItem = {
         title: 'No updates available!',
         body: 'You will automatically receive updates as soon as they are available 🤗'
       })).show();
+
+      track('updates/checked');
```

Comment:
This should rather be more specific for reporting when users click the "Check for Updates" button. Reporting when an update is checked for is not very useful.